class ApplicationError(Exception):
    pass


class NotFound(Exception):
    pass
